packages/rum/src/viewCollection.ts

```diff
@@ -175,3 +198,35 @@ function trackTimings(lifeCycle: LifeCycle, callback: (timings: Timings) => void
   )
   return { stop: stopPerformanceTracking }
 }
+
+interface ViewLoadingState {
+  stop: () => void
+}
+let currentViewLoadingState: ViewLoadingState | undefined
```

Comment:
This shouldn't be needed, because:
* we always call `currentView.end()` (and thus stopping the `trackLoadDuration`) before starting a new view
* even if we wouldn't, it doesn't matter too much if two 'trackLoadDuration' calls are running simultaneously.